docs/_documentations/project-actions.md

```diff
@@ -6,35 +6,40 @@ keywords: run, open, import, show, restart, edit, build, logs, tools, project ac
-- **Launch Debug Session**: If you ended your Node.js debug session accidentally, use this to set up a new debug session. This menu item is only available for Node.js applications that support restart in debug mode.
-
-- **Build**: Initiate a build of your project. This action is not available if a build is already running. 
-- **Disable/Enable Auto Build**: Enable or disable the automatic building of projects. Use this to disable automatic builds if you are making a lot of changes and don't want builds to be triggered until you are done. This option is enabled by default. 
-- **Enable/Disable Inject Metrics**: Enable or disable the injection of open metrics into your application at build time. This option provides monitoring capabilities that are then used by the performance dashboard. This option is disabled by default. 
-
-- **Disable/Enable Project**: Disables or enables the project.
-- **Remove**: This action removes a project from Codewind. You can also optionally delete the project from your workspace and the file system. When you click **Remove**, a confirmation dialog is displayed which has a checkbox for removing the project from the workspace and the file system. The checkbox is unchecked by default.
-- **Refresh**: If the project gets out of sync, use this option to refresh it. To refresh all projects for a connection, right click on the connection and select **Refresh**.
+- **Show all logs**: If log files are available, this action opens all available log files.  
+   - In Eclipse, this action is **Show Log Files>Show All**.
+   - In Eclipse Che and VS Code, this action is **Show all logs**.
+   - In IntelliJ, this action is **Log Files>Show All**.
+- **Hide all logs**: Remove all log files from the **Console** view.
+   - In Eclipse, this action is **Show Log Files>Hide All**.
+   - In Eclipse Che and VS Code, this action is **Hide all logs**.
+   - In IntelliJ, this action is **Log Files>Hide All**.
+- **Manage logs**: In VS Code, this action opens the Command Palette, where you can select which logs you want to see.
+- **Restart in Run Mode**: Restarts the application in run mode. This action is available only on projects that support restart in run mode. 
+- **Restart in Debug Mode**: Restarts the application in debug mode and attaches the debugger. This action is available only on projects that support it. For MicroProfile and Spring projects, a debug session is started automatically. For Node.js projects, you are guided in the setup of a debug session in a Chromium-based browser. For all other project types that support debug, set up a debug session manually. 
+- **Attach Debugger**:  If you accidentally detached the Java debugger or restarted your IDE, use this action to set up a new debug session. This action is available only for Java applications that support restart in debug mode.
```

Comment:
Attach Debugger - The second time you refer to "this action", you can just use "This is..." or "It is..".

Reply:
@sujeilyfonseca It's good for us to avoid using the word "this" by itself and instead qualify it with what "this" is, such as "this action" or "this program." Even though it might sound repetitive, it can help with clarity. :)